code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -87,6 +87,66 @@
 		LAZYADDASSOC(SSjob.dynamic_forced_occupations, new_malf, "AI")
 	return TRUE
 
+//////////////////////////////////////////////
+//                                          //
+//            INTERNAL AFFAIRS              //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/internal_affairs
+	name = "Internal Affairs"
+	antag_flag = ROLE_INTERNAL_AFFAIRS
+	antag_datum = /datum/antagonist/traitor/internal_affairs
+	protected_roles = list(JOB_CAPTAIN, JOB_HEAD_OF_SECURITY, JOB_WARDEN, JOB_SECURITY_OFFICER, JOB_DETECTIVE, JOB_PRISONER)
+	restricted_roles = list(JOB_AI, JOB_CYBORG)
+	required_candidates = 6
+	weight = 4
+	cost = 8
+	scaling_cost = 2
+	requirements = list(8,8,8,8,8,8,8,8,8,8)
```

Comment:
With these stats, it looks like what you've essentially done is just doubled traitor's weight!

Reply:
I've been running this ruleset on 70+ player rounds and have been unable to get more than 6 IAAs at once, I don't know much about ruleset configuration but I was hoping for ~8 since they have to kill eachother rather than the station.